src/main/java/tools/map/making/MapProperties.java

```diff
@@ -83,23 +81,6 @@ public void setColorMap(final Map<String, Color> value) {
     colorMap = value;
   }
 
-  public String outColorMap() {
```

Comment:
Ditto on the `outXXX` methods being accessed reflectively (see #1261 and #1616).

This is the second time code called solely via reflection was removed from this class.  Maybe a big fat comment at the top of this file will help the next dev who comes along with the same idea?